projects/Mallard/src/screens/settings/download-screen.tsx

```diff
@@ -130,6 +130,7 @@ export const DownloadScreen = () => {
                 <List
                     data={fileList}
                     onPress={({ type, path, issue }) => {
+                        console.log(path)
```

Comment:
😶

Reply:
Fixes the no unused vars rule. 😶

Reply:
What happens if you don't specify `path` in the destructuring at all?